core/src/main/scala/org/apache/spark/scheduler/TaskSetManager.scala

```diff
@@ -214,7 +229,8 @@ private[spark] class TaskSetManager(
   private[scheduler] var emittedTaskSizeWarning = false
 
   /** Add a task to all the pending-task lists that it should be on. */
-  private[spark] def addPendingTask(index: Int) {
+  private[spark] def addPendingTask(index: Int,
```

Comment:
args start in next line when you have multi-line args.

Reply:
ok